src/napari_deeplabcut/_widgets.py

```diff
@@ -667,8 +673,16 @@ def settings(self):
     def start_tutorial(self):
         Tutorial(self.viewer.window._qt_window.current()).show()
 
-    def display_shortcuts(self):
-        Shortcuts(self.viewer.window._qt_window.current()).show()
```

Comment:
I'd keep this one-liner here Tim, and we could move the image_path line to the __init__ of the Shortcuts class (since it's hard-coded anyway). Also note that, importantly, the path to the napari_shortcuts.svg needs to be added to the MANIFEST.in file, otherwise people will be missing the file when installing the plugin via pip.

Reply:
DONE added the line back and also added the path to the MANIFEST.in file.